docs/Apps/Data Studio/CCLE Correlation Analysis.md

```diff
@@ -1,11 +1,235 @@
+
+Once all the parameters are selected, execute the component by clicking on *Run Task*.
+
+It will generate three output files:
+
+![Correlation](../../img/ccle/18.png) <center>**Figure 16.** Correlation outputs</center>
+
+
+*   **Pvals Metadata:** Table for p-values of fisher's exact test of selected gene mutation against all the unique values from the metadata column selected.   
+
+*   **Pvals Expression:** Table for p-values of fisher's exact test of selected gene mutation against the binarised expression values of all the genes present in the expression matrix.
+
+*   **Pvals Mutation:** Table for p-values of fisher's exact test of selected gene mutation against the mutation status of all the genes in the mutation matrix.
+
+    
+
+## Differential Expression
+
+This component allows the search for differentially expressed (DE) genes, that is, genes that show differences in expression level between conditions or in other ways are associated with given predictors or responses.
+
+![DE](../../img/ccle/16.png) <center>**Figure 17.** Differential expression parameters</center>
+
+*   **Cohort Column:** Dropdown to select one of the metadata columns.
+
+*   **Cohort A:** Dropdown to select a cohort from the metadata column selected.
+
+*   **Cohort B:** Dropdown to select another cohort from the metadata column selected.
+
+*   **Normalization:** Perform log2 normalization if data is not normalized
+
+*   **Algorithm:** You can select any one of the two algorithms - “limma” or “Unpaired t-test”.
+
+    
+
+*Limma* is an R package for the analysis of gene expression microarray data, especially the use of linear models for analyzing designed experiments and the assessment of differential expression. Limma provides the ability to analyze comparisons between many RNA targets simultaneously in arbitrary complicated designed experiments.
+
+An *unpaired t-test* (also known as an independent t-test) is a statistical procedure that compares the averages/means of two independent or unrelated groups to determine if there is a significant difference between the two.
+
+*   **P-Value Correction:** You can select the “Benjamini-Hochberg” method to correct the p-value for False Discovery Rate or the “Bonferroni” method to correct the p-value for Type I errors.
+
+*   **P-Value Metric:** Plot and calculate significance using the selected metric. P-value is the value returned by the algorithm while Adjusted P-value is the corrected value after applying one of the correction methods above.
+
+*   **P-value threshold:** You can select the appropriate threshold for the selected P-Value metric. P-values lower than this threshold will be marked as Significant
+
+*   **Absolute Log2FC Threshold:** You can select the appropriate fold change threshold. Log2FC values higher than this will be marked as Significant
```

Comment:
2 in subscript. `.` missing. confirm whether it is fc or FC with the rest of the doc